blocks/enrichment/enrichment.js

```diff
@@ -39,11 +39,21 @@ export default async function decorate(block) {
   (await Promise.all(matchingFragments.map((path) => loadFragment(path))))
     .filter((fragment) => fragment)
     .forEach((fragment) => {
-      const section = fragment.querySelector(':scope .section');
-      if (section) {
-        block.closest('.section').classList.add(...section.classList);
+      const sections = fragment.querySelectorAll(':scope .section');
+
+      // If only single section, replace enrichment block with content of section
+      if (sections.length === 1) {
+        block.closest('.section').classList.add(...sections[0].classList);
         const wrapper = block.closest('.enrichment-wrapper');
-        section.childNodes.forEach((child) => wrapper.parentNode.insertBefore(child, wrapper));
+        Array.from(sections[0].children)
+          .forEach((child) => wrapper.parentNode.insertBefore(child, wrapper));
+      } else if (sections.length > 1) {
+        // If multiple section, append them to current section
```

Comment:
> append them to current section

You are inserting them after the current section, not to the current section, right?

Reply:
Yes the sections will be added after the section that contains the enrichment block. Let me update the comment to make this more clear.